README.md

```diff
@@ -1,76 +1,229 @@
+*** Reference links are enclosed in brackets [ ] instead of parentheses ( ).
+*** See the bottom of this document for the declaration of the reference variables
+*** for contributors-url, forks-url, etc. This is an optional, concise syntax you may use.
+*** https://www.markdownguide.org/basic-syntax/#reference-style-links
+-->
+
+
+<!-- TITLE -->
+<div align="center">
+    <a href="https://github.com/Zagrios/bs-manager">
+        <img src="resources/readme/icon.svg" alt="Logo" width="250" height="250">
+    </a>
+    <br><br>
+    <h2><b>BSManager</b></h2>
+    <i>(Beat Saber Manager)</i>
+    <br><br>
+    <span>
+        Download, manage and customize your versions of Beat Saber with a simple click!
+    </span>
+    <br>
+    <span>
+        <a href="https://github.com/Zagrios/bs-manager/issues/new?assignees=Zagrios&labels=bug&template=bug_report.md&title=%5BBUG%5D">Report Bug</a> 
+        ·
+        <a href="https://github.com/Zagrios/bs-manager/issues/new?assignees=Zagrios&labels=enhancement&template=feature_request.md&title=%5BAME%5D">Request Feature</a>
+    </span>
+</div>
 
-<h2 id="having-problems">Having problems?</h2>
-<p>
-See the <a href="https://github.com/Zagrios/bs-manager/issues">open issues</a> for a list of known issues, from there you can create a new issue for the problem you have!  
-Otherwise you can add on to existing issues if one matches what you're encountering.
-</p>
-<h2 id="acknowledgements">Acknowledgement</h2>
+<!-- BADGES -->
+---
+<br>
+<div align="center" >
+    <a href="https://github.com/Zagrios/bs-manager/stargazers"><img src ="https://img.shields.io/github/stars/Zagrios/bs-manager?style=for-the-badge" alt="Stargazers"/></a>
+    <a href="https://github.com/Zagrios/bs-manager/network/members"><img src ="https://img.shields.io/github/forks/Zagrios/bs-manager?style=for-the-badge" alt="Forks"/></a>
+    <a href="https://github.com/Zagrios/bs-manager/blob/master/LICENSE"><img src ="https://img.shields.io/github/license/Zagrios/bs-manager?style=for-the-badge" alt="License"/></a>
+    <a href="https://github.com/Zagrios/bs-manager/graphs/contributors"><img src ="https://img.shields.io/github/contributors/Zagrios/bs-manager?style=for-the-badge" alt="Contributors"/></a>
+    <a href="https://github.com/Zagrios/bs-manager/issues"><img alt="GitHub issues" src="https://img.shields.io/github/issues/Zagrios/bs-manager?style=for-the-badge"></a>
+    <br>
+    <a href="https://discord.gg/uSqbHVpKdV"><img src ="https://img.shields.io/badge/-DISCORD-5865f2?style=for-the-badge&logo=discord&logoColor=ffffff" alt="discord"/></a>
+    <a href="https://mee6.xyz/fr/m/bsmanager"><img src ="https://img.shields.io/badge/-🥰%20Support%20BSM-EC4546?style=for-the-badge" alt="Donation"/></a>
+    <a href="https://discord.gg/uSqbHVpKdV"><img src ="https://img.shields.io/badge/-Twitter-F5F8FA?style=for-the-badge&logo=Twitter" alt="Twitter"/></a>
+    <br>
+    [<a href="#readme-bot">go to bot</a>]
```

Comment:
🟡 bot ? not bottom ?

Reply:
By saying "bot" all the time I forgot that the full word was "bottom" XD

```suggestion
    [<a href="#readme-bot">go to bottom</a>]
```